src/ngx_http_set_hashed_upstream.c

```diff
@@ -25,32 +35,73 @@ ngx_http_set_misc_apply_distribution(ngx_log_t *log, ngx_uint_t hash,
 }
 
 
+uint32_t
+ngx_http_set_misc_bsearch(
+    ngx_http_set_hashed_upstream_consistent_hash_node *nodes, uint32_t node_len,
+    uint32_t target)
+{
+    uint32_t m = 0, n = node_len, l;
```

Comment:
Style: nginx coding style requires each C function local variable to be defined in separate lines.